build.assets/buildbox/thirdparty-libs.mk

```diff
@@ -160,9 +160,9 @@ tp-build-libcbor: fetch-git-libcbor
 # -----------------------------------------------------------------------------
 # openssl
 
-openssl_VERSION = 3.0.13
```

Comment:
@camscale the versions here were all 1 behind other files. I'm not sure if there was a reason for it or if it was just lagging, but I updated it anyway.

Reply:
No worries. I am currently backporting this to the branches, so this won't get backported now but I'll include it in the backports.

Reply:
Be careful to match the versions to the branch, as they differ in some way in almost every branch. We backport security fixes like these, but we rarely (if ever) backport increased functionality.